ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/ActiveContractsServiceIT.scala

```diff
@@ -24,13 +24,16 @@ import com.daml.ledger.test_stable.Test.{
 import io.grpc.Status
 import scalaz.syntax.tag._
 
-class ActiveContractsService(session: LedgerSession) extends LedgerTestSuite(session) {
+import scala.concurrent.ExecutionContext
+
+class ActiveContractsServiceIT extends LedgerTestSuite() {
```

Comment:
Can we lose the unnecessary parentheses after `LedgerTestSuite` in the test classes?

Reply:
Sure.